src/api/analytics/index.tsx

```diff
@@ -0,0 +1,37 @@
+import ReactGA, { InitializeOptions } from 'react-ga'
+import { isMobile } from 'react-device-detect'
+export { pageview as viewPage, event as triggerEvent } from 'react-ga'
+
+function getCustomBrowserType(): string {
+  if (isMobile) {
+    return 'ethereum' in window ? 'mobileWeb3' : 'mobileRegular'
```

Comment:
i don't like using the `in` operator, see why:
```js
window.ethereum = { a: 'b;ah' }
// { a: "blah" }

if ('ethereum' in window) console.log('yay')
"yay" // logs

// set ethereum to `undefined`
window.ethereum = undefined
if ('ethereum' in window) console.log('yay')
"yay" // logs

if (window.ethereum) console.log('yay')
undefined // no log
```

Reply:
okok